prime-dotnet-webapi/Services/DefaultPrivilegeService.cs

```diff
@@ -74,21 +74,48 @@ public async Task AssignPrivilegesToEnrolleeAsync(int enrolleeId, Enrollee enrol
                     }
                 }
 
+                _enrolleeDb.Privileges = this.GetPrivilegesForEnrollee(enrollee);
+                _context.Entry(_enrolleeDb).State = EntityState.Modified;
+
                 await _context.SaveChangesAsync();
             }
         }
 
+        public ICollection<Privilege> GetPrivilegesForEnrollee(Enrollee enrollee)
```

Comment:
Since this hits the databse, should be
`public async Task<ICollection<Privilege>> GetPrivilegesForEnrollee(Enrollee enrollee)` 
(or GetPrivilegesForEnrolleeAsync)